js/modules/organization.src.js

```diff
@@ -0,0 +1,538 @@
+/* *
+ * Organization chart module
+ *
+ * (c) 2018-2019 Torstein Honsi
+ *
+ * License: www.highcharts.com/license
+ */
+
+'use strict';
+
+import H from '../parts/Globals.js';
+var pick = H.pick;
+
+var base = H.seriesTypes.sankey.prototype;
+
+/**
+ * @private
+ * @class
+ * @name Highcharts.seriesTypes.organization
+ *
+ * @augments Highcharts.seriesTypes.sankey
+ */
+H.seriesType(
+    'organization',
+    'sankey',
+    /**
+     * An organization chart is a diagram that shows the structure of an
+     * organization and the relationships and relative ranks of its parts and
+     * positions.
+     *
+     * @sample       highcharts/demo/organization-chart/
+     *               Organization chart
+     * @sample       highcharts/series-organization/horizontal/
+     *               Horizontal organization chart
+     * @sample       highcharts/series-organization/borderless
+     *               Borderless design
+     * @sample       highcharts/series-organization/center-layout
+     *               Centered layout
+     *
+     * @extends      plotOptions.sankey
+     * @excluding    allowPointSelect, curveFactor
+     * @since        7.1.0
+     * @product      highcharts
+     * @optionparent plotOptions.organization
+     */
+    {
+        /**
+         * The border color of the node cards.
+         * @type {Highcharts.ColorString}
+         */
+        borderColor: '${palette.neutralColor60}',
+        /**
+         * The border radius of the node cards.
+         */
+        borderRadius: 3,
+        /**
+         * Radius for the rounded corners of the links between nodes.
+         *
+         * @sample   highcharts/series-organization/link-options
+         *           Square links
+         */
+        linkRadius: 10,
+        borderWidth: 1,
+        dataLabels: {
+            /**
+             * A callback for defining the format for _nodes_ in the
+             * organization chart. The `nodeFormat` option takes precedence over
+             * `nodeFormatter`.
+             *
+             * In an organization chart, the `nodeFormatter` is a quite complex
+             * function of the available options, striving for a good default
+             * layout of cards with or without images. In organization chart,
+             * the data labels come with `useHTML` set to true, meaning they
+             * will be rendered as true HTML above the SVG.
```

Comment:
Could we also inform users that `allowHTML` for exporting should  be enabled manually?